nck/readers/mytarget_reader.py

```diff
@@ -16,51 +16,36 @@
 # along with this program; if not, write to the Free Software Foundation,
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 import click
-import nck.helpers.api_client_helper as api_client_helper
-from nck.clients.api_client import ApiClient
 from nck.commands.command import processor
 from nck.readers.reader import Reader
 from nck.streams.json_stream import JSONStream
 from nck.utils.args import extract_args
 import requests
-from typing import Dict
-from nck.utils.secret_retriever import get_secret
-from botocore.config import Config
+from typing import Dict, List, Any
 import logging
-import json
-import boto3
 import itertools
+import time
 
 
 @click.command(name="read_mytarget")
 @click.option("--mytarget-client-id", required=True)
 @click.option("--mytarget-client-secret", required=True)
 @click.option("--mytarget-mail", required=True)
 @click.option("--mytarget-agency", required=True)
+@click.option("--mytarget-refresh-token", required=True)
+@click.option("--mytarget-start-date", type=click.DateTime())
+@click.option("--mytarget-end-date", type=click.DateTime())
+@click.option(
+    "--mytarget-date-format",
+    default="%Y-%m-%d",
+    help="And optional date format for the output stream. "
+    "Follow the syntax of https://docs.python.org/3.8/library/datetime.html#strftime-strptime-behavior",
+)
```

Comment:
For now, we use the JSONStream, so there is no option to customize date format. So, is this parameter still relevant?